.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,18 @@
+<!--
+- Please fill in this template according to the PR you're about to submit
+- Features and deprecations must be submitted against the master branch
+- Bug fixes must be submitted against the lowest branch where they apply
+  (lowest branches are regularly merged to upper ones so they get the fixes too)
```

Comment:
Can you add full stops at the end of the sentences? I was reading it like one big sentence and it felt off